data/events.toml

```diff
@@ -0,0 +1,63 @@
+# eventOrder : integer (used to order events ascending order)
+# eventType : must be "upcoming" or "past"
+# featured : boolean (if true, the event will be shown on home pade NOTE: We only display the "Featured event section" if there are more than three events to display.)
+
+
+[[event]]
+    dataTime = "Thu Nov 17, 2022, 8:30 PM IST"
+    title = "Bayesian Modeling in Biotech: Using PyMC to Analyze Agricultural Data."
+    place = "Online"
+    description = "In this panel discussion we will discuss why Bayesian modeling is such a powerful tool for solving problems in biotechnology. As experiments are often complex it is important to build custom and causal models that accurately represent the structure of the experiment in the statistical model. In this panel discussion we will discuss why Bayesian modeling is such a powerful tool for solving problems in biotechnology. As experiments are often complex it is important to build custom and causal models that accurately represent the structure of the experiment in the statistical model."
```

Comment:
Is there a way to have the box always expanded rather than expand over scrollover

<img width="1328" alt="image" src="https://user-images.githubusercontent.com/7213793/211214969-52072799-89f8-44a0-83ce-16b0f24d7b9b.png">

Reply:
I had the same thought at first. I then consider that the description may be too long, affecting the consistent size of the cards, so I create them to show few lines and expand on user interaction.
but I have now corrected them.

However, this effect remains for event headings of featured events; it will only show the first two lines and expand the rest based on user interaction(you can check one of the featured event card); let me know if you want to expand that as well.

Reply:
got it, I trust you thought about it more than me. Thank you for explaining it. No need for a change

Reply:
And thanks for considering all the options